packages/host-portal-app/src/components/remote-view.tsx

```diff
@@ -10,55 +9,69 @@ import React, {
 import { Manifest, View } from '../types';
 
 const loading = Symbol('loading');
-type MicroFrontendState = Record<string, React.ComponentType | typeof loading>;
 
-const views = createContext<
-  [
+type MicroFrontendState = Record<string, React.ComponentType | typeof loading>;
+type GlobalStylesheetSet = Set<string>;
```

Comment:
As these are called `styleImports` in the modular library, might it be more clear to use the same name throughout?

Reply:
n/a anymore, see https://github.com/cristiano-belloni/modular-microfrontends/pull/7#discussion_r1014803846